src/components/select/filterable-select/filterable-select.stories.mdx

```diff
@@ -637,6 +638,76 @@ If there is no `id` prop specified on an object, then the exact objects will be
   </Story>
 </Canvas>
 
+#### New designs validation
+
+<Preview>
```

Comment:
There is a missing import here for `Preview`

Reply:
Yeah these should have been `Canvas` not `Preview` - copy and paste error!